cfme/fixtures/v2v_fixtures.py

```diff
@@ -0,0 +1,522 @@
+       In 5.9 all rhev hosts and tagged with tags V2V - Transformation Host=t
+       and V2V - Transformation Method=vddk/ssh.These tags are automatically removed
+       once the provider is deleted.
+
+       In 5.10 rails console commands are run to configure all the rhev hosts.
+
+    Args:
+        appliance:
+        transformation_method : vddk or ssh as per test requirement
+        rhev_hosts: number of hosts in rhev to configure for conversion
+    """
+
+    tag1 = appliance.collections.categories.instantiate(
+        display_name="V2V - Transformation Host *"
+    ).collections.tags.instantiate(display_name="t")
+    tag2 = appliance.collections.categories.instantiate(
+        display_name="V2V - Transformation Method"
+    ).collections.tags.instantiate(display_name=transformation_method)
+
+    # Delete all prior conversion hosts otherwise it creates duplicate entries
+    appliance.ssh_client.run_rails_command("'ConversionHost.delete_all'")
+
+    for host in rhev_hosts:
+        # if _tag_cleanup() returns True, means all tags were removed
+        if _tag_cleanup(host, tag1, tag2):
+            # so we call add_tags to add only required tags
+            host.add_tags(tags=(tag1, tag2))
+            # set conversion host via rails console
+            if appliance.version >= "5.10":
+                if host.provider.one_of(RHEVMProvider):
+                    resource_type = "Host"
+                else:
+                    resource_type = "VmOrTemplate"
+                appliance.ssh_client.run_rails_command(
+                    "'r = Host.find_by(name:{host});\
+                c_host = ConversionHost.create(name:{host},resource_id:r.id,resource_type:{type});\
+                c_host.{method}_transport_supported = true;\
+                c_host.save'".format(
+                        host=json.dumps(host.name),
+                        type=json.dumps(resource_type),
+                        method=transformation_method.lower(),
+                    )
+                )
+
+
+def set_conversion_instance_for_osp(appliance, transformation_method, osp_provider):
+    """
+    Args:
+        appliance
+        transformation_method: vddk or ssh
+        osp_provider: OSP
+
+        Rails console command
+        ====================
+        res = Vm.find_by(name: 'my_osp_instance')
+        conversion_host = ConversionHost.create(name: res.name, resource: res)
+        conversion_host.vddk_transport_supported = true
+        conversion_host.save
+    """
+    # Delete all prior conversion hosts otherwise it creates duplicate entries
+    appliance.ssh_client.run_rails_command("'ConversionHost.delete_all'")
+    conversion_instance = osp_provider.data.get('conversion_instance1', None)
+
+    resource_type = "VmOrTemplate"
+    appliance.ssh_client.run_rails_command(
+        "'r = Vm.find_by(name:{vm});\
+    c_host = ConversionHost.create(name:{vm},resource_id:r.id,resource_type:{type});\
```

Comment:
@sshveta  you are using deprecated rails commands here, no need to specify `resource_id` and `resource_type` now.
We should add 

```
res = Vm.find_by(name: 'my_osp_instance')
conversion_host = ConversionHost.create(name: res.name, resource: res)
conversion_host.vddk_transport_supported = true # appropriate transport method
conversion_host.save
```

Please update it accordingly to RHV's conversion hosts as well, only change is, 
```
res = Host.find_by(name: 'my_rhv_host')
```